libs/media-player/src/lib/shared/constants.ts

```diff
@@ -0,0 +1,71 @@
+export const EDITOR_SOUND_BAR_HEIGHT_IN_PIXELS = 40;
```

Comment:
I'd prefer to see the entire ruler \ canvas \ range bar in a single directory within shared.